compiler/ballerina-lang/src/main/resources/compiler.properties

```diff
@@ -646,6 +646,9 @@ error.annotation.invalid.type=\
   annotation declaration requires a subtype of ''true'', ''map<value:Cloneable>'' or ''map<value:Cloneable>[]'', but \
   found ''{0}''
 
+error.greater.than.signed.int32.max.array.sizes.not.yet.supported=\
+  greater than signed int32 max array sizes not yet supported
```

Comment:
Shall we change the error message a bit? 
`array size greater than int:Signed32 max is not yet supported`
@gimantha @rdhananjaya WDYT?

Reply:
We also have this https://github.com/ballerina-platform/ballerina-spec/blob/master/lang/lib/int.bal#L59 [1]
But one thing to keep in mind is largest supported size is 10 less than the [1], I'm not sure how to word that though.

Reply:
Just saw the other comment re: mentioning the exact number, but IMO it is OK since it conveys the exact value that can be used as the maximum.

Reply:
Btw, even when we did support this, wouldn't the maximum size that can be specified as the size be `9223372036854775807`? 

We should give a different (invalid size) error for such scenarios, right? Since they will never be allowed?